tests/thread_msg_block_race/main.c

```diff
@@ -84,7 +85,7 @@ int main(void)
     timer_init(TIMER_DEV(0), TIMER_FREQ, _timer, NULL);
     random_init(timer_read(TIMER_DEV(0)));
     puts("Test is \"successful\" if it runs forever without halting\n"
-         "on any of the assertion in this file\n");
+         "on any of the expection in this file\n");
```

Comment:
Search & replace error, I believe.

Reply:
> Search & replace error, I believe.

Not in this case. I wasn't really sure. At least the right word would be `expectation`. I can change it back or correct. What would you prefer?

Reply:
I think its semantics at this point, as an assertion is an expectation, but if you prefer an expect-base word, I would prefer correct english (pretty sure there is no word `expection` unless its a typo of `exception` ;-))